### PR TITLE
Remove style-related rules

### PR DESCRIPTION
**Pull request by zautumnz**
## Change Type

* [ ] Feature
* [x] Chore
* [ ] Bug Fix

## Change Level

* [x] major
* [ ] minor
* [ ] patch

## Further Information (screenshots, bug report links, etc)

At Jane we always use Prettier. It seems a little silly to maintain extra style-related rules. Even if for some reason we got rid of Prettier, we'd probably go for something like Standard (which these style rules are based on). I think it probably makes sense to just delete these.

### CODE REVIEW

File: rules/style.js [hunk 1]
```diff
@@ -1,78 +0,0 @@
...
-  'arrow-spacing': [2, { before: true, after: true }],
-  'block-spacing': [2, 'always'],
-  'brace-style': [2, '1tbs', { allowSingleLine: true }],
-  'comma-spacing': [2, { before: false, after: true }],
-  'comma-style': [2, 'last'],
-  'dot-location': [2, 'property'],
-  'func-call-spacing': [2, 'never'],
-  'generator-star-spacing': [2, { before: true, after: true }],
-  indent: [2, 2, { SwitchCase: 1 }],
-  'jsx-quotes': [2, 'prefer-double'],
```
[the remark this answers is not in the document]
Comment (zacanger) [reply rules/style.js]: Yep

**Review by TomPridham — APPROVED** [review]
lgtm then

**Comment by lukeschunk** [comment]
seems good to me

**Comment by jhelst** [comment]
:shipit:

**Pull request merged**